netlify.toml

```diff
@@ -1,6 +1,3 @@
 [build]
   publish = "public"
   command = "npm run build"
-[build.environment]
```

Comment:
I removed this in order to see if it helps to get netlify using npm and fix some problems during the build process.